src/main/java/org/checkerframework/checker/builder/TypesafeBuilderAnnotatedTypeFactory.java

```diff
@@ -1,28 +1,29 @@
 package org.checkerframework.checker.builder;
 
-import com.sun.source.tree.ExpressionTree;
-import com.sun.source.tree.MethodInvocationTree;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Set;
+import com.google.auto.value.AutoValue;
+import com.sun.source.tree.*;
```

Comment:
We stylistically prefer no wildcard import in Checker Framework code, so this will be rejected if we try to move it into the main repository. I acknowledge I don't have a lint rule turned on for it right now in this repo, though, so if you want to leave it it's fine.